Dockerfile

```
@@ -32,11 +32,11 @@ RUN <<-EOT
 	sudo git config --system --add safe.directory "*"
 	
 	sudo mkdir -p /tmp/docker-install
-	sudo curl -fsSL "https://download.docker.com/linux/static/stable/$(uname -m)/docker-24.0.7.tgz" | sudo tar -xz -C /tmp/docker-install
+	sudo curl -fsSL "https://download.docker.com/linux/static/stable/$(uname -m)/docker-28.2.2.tgz" | sudo tar -xz -C /tmp/docker-install
 	sudo mv /tmp/docker-install/docker/docker /usr/local/bin/
 	sudo rm -rf /tmp/docker-install
 	sudo mkdir -p /usr/local/lib/docker/cli-plugins
-	sudo curl -fsSL "https://github.com/docker/compose/releases/download/v2.24.6/docker-compose-linux-$(uname -m)" -o /usr/local/lib/docker/cli-plugins/docker-compose
+	sudo curl -fsSL "https://github.com/docker/compose/releases/download/v2.37.0/docker-compose-linux-$(uname -m)" -o /usr/local/lib/docker/cli-plugins/docker-compose
```

```suggestion
	sudo curl -fsSL "https://github.com/docker/compose/releases/latest/download/docker-compose-linux-$(uname -m)" -o /usr/local/lib/docker/cli-plugins/docker-compose
```

According to https://github.com/docker/compose/releases/download/v2.37.0/docker-compose-linux-$(uname -m)